refactor/src/Config.py

```diff
@@ -73,12 +73,19 @@ def __getattr__(self, name):
 
 
 class RuntimeConfig:
-    """Simulation parameters and constants"""
+    """@DynamicAttrs Simulation parameters and constants"""
 
     def __init__(self, hyperparam_file):
         with open(hyperparam_file) as f:
             data = json.load(f)
 
+        # FIXME: make pycharm/mypy perform detection of these vars
+        # Or alternatively manually set the vars
```

Comment:
Let's set them manually - will solve pycharm problems and will possibly allow for better validation in the future (plus there's not that many of them).